src/styles/blocks/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  box-sizing: border-box;
```

Comment:
better add box-sizing: border-box in `*` selector

Reply:
not fixed

Reply:
If I remove these values and make them in * , the header will not display correctly.